pipeline/frontend/drone_compatibility.go

```diff
@@ -15,22 +15,22 @@
 package frontend
 
 // setDroneEnviron set dedicated to DroneCI environment vars as compatibility layer
-func (m *Metadata) setDroneEnviron(env map[string]string) {
+func setDroneEnviron(env map[string]string) {
```

Comment:
we should use metadata directly and not some env var!

Reply:
-> if you dont like change this, I'll create a pull sepearte first before we merge - just tell me